### ISSUE TITLE
ktest - report which files are excluded by exclude patterns

### ISSUE THREAD

**Issue by daejunpark** [kframework/k-legacy#1422]
The `exclude` pattern might be confusing for some reasons such as #1421, #232.

It would be useful to report explicitly which files are excluded by `exclude` patterns at least in the `verbose` mode.

**Comment by osa1** [comment]
Good idea. It's been a while since last time I built and looked at the K code base, but I'll see what I can do :-) (probably during the weekend)
